src/base/xserver_interface.h

```diff
@@ -173,7 +173,11 @@ extern bool logtb_has_updated (THREAD_ENTRY * thread_p);
 
 
 extern BTID *xbtree_add_index (THREAD_ENTRY * thread_p, BTID * btid, TP_DOMAIN * key_type, OID * class_oid, int attr_id,
-			       int unique_pk, long long num_oids, long long num_nulls, long long num_keys);
+			       int unique_pk, long long num_oids, long long num_nulls, long long num_keys
+#if defined(SUPPORT_DEDUPLICATE_KEY_MODE)
+			       , int deduplicate_key_attr_pos
```

Comment:
It's trivial.
The btree_add_index and sbtree_add_index functions use the name deduplicate_key_pos,
while xbtree_add_index uses the name deduplicate_key_attr_pos.
Is there any reason why they use different names?

Reply:
There is no particular reason.
I will unify it with deduplicate_key_pos.